pyproject.toml

```
@@ -719,6 +720,9 @@ ignore = [
 
     # temporarily disabled
     "PT019",
+    "PYI041",
+    "PYI036",
+    "PYI024",
```

```suggestion
    "PYI024", # Use typing.NamedTuple instead of collections.namedtuple
    "PYI036",
    "PYI041",
```
Could you sort these? Furthermore, I would suggest to add a comment for `PYI024` explaining the error code. It might take longer to convert all `namedtuples`.